app/views/buyers/accounts/_search_results.html.slim

```diff
@@ -50,7 +50,7 @@ table class="data" id="buyer_accounts"
           td class="plan" = h account.bought_account_plan.try(:name)
 
         - if can? :see, :multiple_applications
-          td = link_to (current_user.accessible_cinstances.bought_by(account).count), admin_buyers_account_applications_path(account)
+          td = link_to account.bought_cinstances_count, admin_buyers_account_applications_path(account)
```

Comment:
I am not sure if it is the same 🤔 `account.bought_cinstances_count` is the number of bought cinstances. `current_user.accessible_cinstances.bought_by(account).count)` is the number of bought **accessible** cinstances. What I do not know is a non-accessible cinstances can be bought at any point in time.

Reply:
That's a bit worrying because the whole point of the new column is to provide Portafly with this very value.

Reply:
I think that it is correct but I am not 100% sure so I left the comment in case that someone can be sure of that 😄 
I believe that only accessible plans can be bought, what I do not know for sure if a plan that has been accessible, and has bought plans can become inaccessible later on. I hope not for our mental health 😛

Reply:
Strictly, it's not the same. The access is granted to a user for a given service (or set of services), not for buyer accounts or applications. Consider the following:
- current_user has access to services A and B, but not C
- account X bought cinstances of A and C (1 of each)
- `current_user.accessible_cinstances.bought_by(account_x).count` should be 1
- `account_x. bought_cinstances_count` will return 2